components/file_upload/file_upload.jsx

```diff
@@ -617,10 +615,10 @@ export default class FileUpload extends PureComponent {
                             this.setState({menuOpen: false});
                         }}
                     >
-                        <button className='style--none'>
+                        <a>
```

Comment:
will this cause the desktop app to show the link in the status bar?

Reply:
I don't think so but I'm not 100% sure. Either way this is just going back to the behaviour we had before